extensions/wikia/PortableInfobox/tests/PortableInfoboxDataServiceTest.php

```diff
@@ -127,41 +184,74 @@ public function testConstructor() {
 		$this->assertEquals( [ ], $result );
 	}
 
-	protected function getInfoboxData() {
-		return [ [ 'data' => [ [ "type" => "data",
-								 "data" => [
-									 "value" => "AAAA",
-									 "label" => "BBBB"
-								 ]
-							   ], [ "type" => "image",
-									"data" => [ [
-										"key" => "Test.jpg",
-										"alt" => null,
-										"caption" => null,
-									] ]
-							   ], [ "type" => "image",
-									"data" => [ [
-										"key" => "Test2.jpg",
-										"alt" => null,
-										"caption" => null
-									] ] ] ] ],
-				 [ 'data' => [ [ "type" => "image",
-								 "data" => [ [
-									 "key" => "Test2.jpg",
-									 "alt" => null,
-									 "caption" => null
-								 ] ] ] ] ] ];
+	protected function getInfoboxPageProps() {
+		return [
+			[
+				'parser_tag_version' => PortableInfoboxParserTagController::PARSER_TAG_VERSION,
+				'data' => [
+					[
+						'type' => 'data',
+						'data' => [
+							'value' => 'AAAA',
+							'label' => 'BBBB'
+						]
+					],
+					[
+						'type' => 'image',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "image" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)